packages/lesswrong/lib/collectionUtils.ts

```diff
@@ -198,9 +198,14 @@ export function schemaDefaultValue<T extends DbObject>(defaultValue: any): Parti
   }
 }
 
-export function addUniversalFields<T extends DbObject>({ collection, schemaVersion=1 }: {
+export function addUniversalFields<T extends DbObject>({
+  collection,
+  schemaVersion = 1,
+  createdAtOptions = {},
+}: {
   collection: CollectionBase<T>,
   schemaVersion?: number
+  createdAtOptions?: Record<string, any>,
```

Comment:
Can we type this to at least be limited to only those fields (and their respective types) we're currently using it for, if not the full list of permissible options?

Reply:
Looks like `editableBy`, `insertableBy`, `viewableBy`, and `canRead`.